deploy/.env.example

```
@@ -12,8 +12,23 @@ DB_URL=jdbc:postgresql://db:5432/gachi
 DB_USERNAME=gachi
 DB_PASSWORD=
 
+# SMTP (required when AUTH_EMAIL_NOOP_ALLOWED=false)
+SPRING_MAIL_HOST=
+SPRING_MAIL_PORT=587
+SPRING_MAIL_USERNAME=
+# 로컬 개발 편의를 위한 fallback 비밀번호(운영/공용 환경에서는 비워두는 것을 권장)
+SPRING_MAIL_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SPRING_MAIL_PASSWORD key should go before the SPRING_MAIL_PORT key

(UnorderedKey)